### DISCUSSION TITLE
Funding surveydown for community development

### DISCUSSION THREAD

**Discussion by jhelvy** [surveydown-dev/surveydown#158] [Q&A]
### Description

This project has reached a level of momentum that we felt we should consider applying for funding to continue building out the community of users and developers. This discussion is an effort to try and transparently search for funding opportunities that we might consider applying for. I would appreciate any thoughts / suggestions anyone out there has on this topic. Feel free to comment on funding opportunities that you might know of and / or your thoughts on what we should be focused on. If you have any suggestions on _how_ we might go about this endeavor (obtaining funding or building a community), please feel free to let us know here! 

To get us started, here's a list of opportunities to consider:

- [NSF Pathways to Enable Open-Source Ecosystems (POSE)](https://new.nsf.gov/funding/opportunities/pathways-enable-open-source-ecosystems-pose)
- [NSF The Safety, Security, and Privacy of Open-Source Ecosystems (Safe-OSE)](https://new.nsf.gov/funding/opportunities/safe-ose-safety-security-privacy-open-source-ecosystems)
- [NSF CyberInfrastructure for Public Access and Open Science (CI PAOS)](https://new.nsf.gov/funding/opportunities/ci-paos-cyberinfrastructure-public-access-open-science)
- [NIH Building Sustainable Software Tools for Open Science (R03 Clinical Trial Not Allowed)](https://grants.nih.gov/grants/guide/rfa-files/RFA-OD-24-010.html)
- [NIH Research Software Engineer (RSE) Award (R50 Clinical Trials Not Allowed)](https://grants.nih.gov/grants/guide/rfa-files/RFA-OD-24-011.html)
- [NASA Open Science Funding Opportunities](https://science.nasa.gov/open-science/nasa-open-science-funding-opportunities/)
- [The Navigation Fund: Open Science](https://www.navigation.org/grants/open-science) 
- [Software Sustainability Institute](https://www.software.ac.uk/) (they have a research software healthcheck program that might be helpful).
- [Invest in Open Infrastructure](https://investinopen.org/)

**Comment by jhelvy** [comment]
Looking at these options, I think the [Navigation Fund](https://www.navigation.org/grants/open-science) and the [NIH program](https://grants.nih.gov/grants/guide/rfa-files/RFA-OD-24-010.html) look like potentially good fits for actually continuing the software development. The Navigation Fund has a "[introduce yourself](https://www.navigation.org/grants/open-science/introduce-yourself)" page where I think we could write a compelling narrative to introduce surveydown. I might try this after we get a draft paper written. 

The NSF POSE program I see as a better fit for supporting the build out of the community of users / devs. I would also see this as an important opportunity that surveydown might be well-suited for.

**Comment by david-lippert-gwu** [comment]
I really like having this discussion in the open.  Great idea.

The GW OSPO is excited to support surveydown.  Here are some additional resources and ideas that we hope are helpful.

Community building
- github has some nice [open source guides](https://opensource.guide/) to support projects that are looking to grow
- This [repo-scaffolder/maturity-model-tiers matrix](https://github.com/DSACMS/repo-scaffolder/blob/main/maturity-model-tiers.pdf) has a nice view of various markdown files that are relevant for projects based on their maturity.
- This [All Contributors Bot](https://allcontributors.org/) is a great way to automate the recognition of all contributors (including non coders) who are supporting your community.

Internally at GW I plan to reach out to IT and the Research Technology Services teams to see if they are able to join the surveydown community to ensure there is a plan to support and sustain your project as it becomes more mature.  Also, the OSPO is excited to partner with your team on the NSF or other grants.

Externally, I plan to talk with other universities and research entities involved in the [CURIOSS](https://curioss.org/) (Academic OSPO network) to see if they might be interested in also joining the surveydown community.

**Comment by byrdnick** [comment]
[NSF Research Infrastructure in the Social and Behavioral Sciences (RISBS)](https://www.nsf.gov/funding/opportunities/risbs-research-infrastructure-social-behavioral-sciences)?

[NSF Human Networks and Data Science (HNDS)](https://www.nsf.gov/funding/opportunities/hnds-human-networks-data-science)?

**Reply by jhelvy** [reply]
The RISBS description sounds like a potentially very good fit by its description, though the recent awarded grants page shows just 9 grants, and none are for software development (despite that being exactly what it says in the program description). I might email the PM and see what they think. HNDS looks like a bit of a stretch too unless we have a sort of basic research task that we go after and make the justification that developing surveydown is an integral part of achieving the research task (e.g., we need to develop it further since there are no other suitable tools or something like that).

**Comment by bastianolea** [comment]
Sorry for replying in an old thread, but in the meantime, why not enable GitHub Sponsors or something similar? I think many users would be glad to give out a couple bucks to the project!

**Reply by jhelvy** [reply]
I really appreciate it, but what we would need to make surveydown more sustainable is something more significant. I am looking for funding to support a full-time appointment on it, such as a postdoc position. I don't think the community is large enough (yet) to support it at that level. Eventually it would be great if we could find a more long-term model to support it, such as a pay-for-premium service model (e.g., how supabase works), but there's a lot of work needed to get it into a shape like that, which is what I think we could do with something like a postdoc dedicated to the project for a year or two.
